### ISSUE TITLE
Gitslackin request new reviewer

### ISSUE THREAD

**Issue by kylebrownnc** [James-Firth/git-slackin#65]
In slack it'd be nice to ask gitslackin to reassign a PR that I'm assigned to, to someone else who is _not_ benched. @gitslackin <repo>/<#pr> or something like that?

It'd be useful for when you need to unbench yourself and are not sure of who is also benched to avoid the same problem that initally occured. 

alternative to this is to allow all users to see the bench list and then they can manually reassign accordingly

**Comment by James-Firth** [comment]
Yeah this seems like a useful feature for when you need to bench yourself after you go "Oh crud I forgot to bench myself". 

Still have a handful of things for the multi-team milestone, but I could see this coming shortly after.

**Comment by kylebrownnc** [comment]
Further context: This is to avoid `!pr-pls-one` spam
